src/bp/ui-admin/src/Pages/Bot/Details.jsx

```diff
@@ -1,26 +1,25 @@
+import { Button, Intent, Callout } from '@blueprintjs/core'
```

Comment:
I think it's been a long time, what do you think about removing this page from the admin panel, and redirecting the user on the studio instead ? That warning was there long enough, I think we can go forward with the change

Reply:
Let's do it. 👍

Reply:
Ok!